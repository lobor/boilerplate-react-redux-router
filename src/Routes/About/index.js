import React from 'react'
import { Row, Box, H1 } from 'components'

export default class About extends React.Component{
	render(){
		return (
			<Row>
				<Row>
					<H1>
						Anteponere illud amicitiae esse veterrima
					</H1>
				</Row>
				<Row>
					<Box>
						Cum haec taliaque sollicitas eius aures everberarent expositas semper eius modi rumoribus et patentes, varia animo tum miscente consilia, tandem id ut optimum factu elegit: et Vrsicinum primum ad
						se venire summo cum honore mandavit ea specie ut pro rerum tunc urgentium captu disponeretur concordi consilio, quibus virium incrementis Parthicarum gentium a arma minantium impetus frangerentur.
					</Box>
					<Box>
						Cum haec taliaque sollicitas eius aures everberarent expositas semper eius modi rumoribus et patentes, varia animo tum miscente consilia, tandem id ut optimum factu elegit: et Vrsicinum primum ad
						se venire summo cum honore mandavit ea specie ut pro rerum tunc urgentium captu disponeretur concordi consilio, quibus virium incrementis Parthicarum gentium a arma minantium impetus frangerentur.
					</Box>
				</Row>
				<Row flex>
					<Box style={{ height: "400px", backgroundColor: "#EEE" }} />
				</Row>
				<Row>
					<Box>
						Cum haec taliaque sollicitas eius aures everberarent expositas semper eius modi rumoribus et patentes, varia animo tum miscente consilia, tandem id ut optimum factu elegit: et Vrsicinum primum ad
						se venire summo cum honore mandavit ea specie ut pro rerum tunc urgentium captu disponeretur concordi consilio, quibus virium incrementis Parthicarum gentium a arma minantium impetus frangerentur.
					</Box>
				</Row>
			</Row>
		);
	}
}
