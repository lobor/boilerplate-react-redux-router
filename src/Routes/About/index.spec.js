import React from 'react'
import renderer from 'react-test-renderer'

import About from './index'

describe('Test route About', () => {
	const tree = renderer.create(<About />).toJSON()

	test('render About', () => {
	  expect(tree).toMatchSnapshot()
	})
})
