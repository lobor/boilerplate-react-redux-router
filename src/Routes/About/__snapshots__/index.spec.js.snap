// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test route About render About 1`] = `
<div
  className="sc-htpNat bBHonn"
>
  <div
    className="sc-htpNat bBHonn"
  >
    <h1
      className="sc-bdVaJa kwHLXg"
    >
      Anteponere illud amicitiae esse veterrima
    </h1>
  </div>
  <div
    className="sc-htpNat bBHonn"
  >
    <div
      className="sc-bwzfXH UDjhZ"
    >
      Cum haec taliaque sollicitas eius aures everberarent expositas semper eius modi rumoribus et patentes, varia animo tum miscente consilia, tandem id ut optimum factu elegit: et Vrsicinum primum ad se venire summo cum honore mandavit ea specie ut pro rerum tunc urgentium captu disponeretur concordi consilio, quibus virium incrementis Parthicarum gentium a arma minantium impetus frangerentur.
    </div>
    <div
      className="sc-bwzfXH UDjhZ"
    >
      Cum haec taliaque sollicitas eius aures everberarent expositas semper eius modi rumoribus et patentes, varia animo tum miscente consilia, tandem id ut optimum factu elegit: et Vrsicinum primum ad se venire summo cum honore mandavit ea specie ut pro rerum tunc urgentium captu disponeretur concordi consilio, quibus virium incrementis Parthicarum gentium a arma minantium impetus frangerentur.
    </div>
  </div>
  <div
    className="sc-htpNat hfAtJp"
  >
    <div
      className="sc-bwzfXH UDjhZ"
      style={
        Object {
          "backgroundColor": "#EEE",
          "height": "400px",
        }
      }
    />
  </div>
  <div
    className="sc-htpNat bBHonn"
  >
    <div
      className="sc-bwzfXH UDjhZ"
    >
      Cum haec taliaque sollicitas eius aures everberarent expositas semper eius modi rumoribus et patentes, varia animo tum miscente consilia, tandem id ut optimum factu elegit: et Vrsicinum primum ad se venire summo cum honore mandavit ea specie ut pro rerum tunc urgentium captu disponeretur concordi consilio, quibus virium incrementis Parthicarum gentium a arma minantium impetus frangerentur.
    </div>
  </div>
</div>
`;
